Weather/Weather/Scenes/ListCityWeather/ListCityWeatherViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  CityManagermentScreenViewController.swift
+//  Weather
+//
+//  Created by minh duc on 2/25/19.
+//  Copyright © 2019 minhduc. All rights reserved.
+//
+
+import UIKit
+import Reusable
+
+final class ListCityWeatherViewController: UIViewController, StoryboardSceneBased {
+    @IBOutlet private weak var listCityCollectionView: UICollectionView!
+    
+    private struct Constant {
+        static let itemSize = CGSize(width: Constants.widthScreen/2, height: Constants.widthScreen/2)
+    }
+    
+    static let sceneStoryboard = UIStoryboard(name: Constants.nameStoryboard, bundle: nil)
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configCollectionView()
+    }
+    
+    private func configCollectionView() {
+        listCityCollectionView.do {
+            $0.delegate = self
+            $0.dataSource = self
+            $0.register(cellType: ListCityWeatherCell.self)
+        }
+    }
+    
+    @IBAction private func addItemButtonTapped(_ sender: UIButton) {
```

Comment:
```suggestion
    @IBAction private func handleAddItemButton(_ sender: UIButton) {
```
nên đặt tên như này sẽ clear hơn